hack/tests/sanity-check.sh

```diff
@@ -14,4 +14,4 @@ make install
 go run ./hack/generate/samples/generate_all.go
 
 # Make sure repo is still in a clean state.
-git diff --exit-code
+git diff --exit-code -- . ':!testdata/go/memcached-operator/bin/*'
```

Comment:
This can be removed for now. We can address in a follow-up.

Reply:
OK.